contracts/contracts/VirtualBalanceRewardPool.sol

```diff
@@ -77,10 +77,15 @@ abstract contract VirtualBalanceWrapper {
  *          For example the Booster sends veCRV fees (3Crv) to a VirtualBalanceRewardPool
  *          which tracks the virtual balance of cxvCRV stakers and distributes their share
  *          of 3Crv rewards
+ *          
+ *          AuraFinance
+ *          - rewardToken: is the StashToken a non-ERC20 compliant contract, the ERC20 token
+ *           is the StashToken.baseToken.
  */
 contract VirtualBalanceRewardPool is VirtualBalanceWrapper {
     using SafeERC20 for IERC20;
     
+    /// @notice The StashToken address, it is not an ERC20 it is a wrapper of the reward token.
     IERC20 public immutable rewardToken;
```

Comment:
Let's change this interface to IStashToken ?

Also on the line above write " WARNING " in capital letters and try to get attn